src/scripts/genXplatEventing.py

```diff
@@ -340,18 +339,7 @@ def generateClrallEvents(eventNodes,allTemplates):
         fnbody.append(lindent)
         fnbody.append("ULONG status = EventPipeWriteEvent" + eventName + "(" + ''.join(line) + ");\n")
         fnbody.append(lindent)
-        fnbody.append("if(XplatEventLogger::IsEventLoggingEnabled())\n")
```

Comment:
I think you should call the specific EventXplatEnabled function here and if it returns true, then call FireEtXplat*.  Basically replace the call to XplatEventLogger::IsEventLoggingEnabled() with the call to the call to EventXplatEnabled*.

Reply:
I'm currently not calling that because if the check isn't inlined, we might as well call the FireEtXplat directly. The first thing it does is check if the event is enabled.

Reply:
Ok, I think that's fine.